scripts/2.TCGA-process.py

```diff
@@ -232,37 +244,44 @@
 expr_df.shape
 
 
-# In[20]:
+# In[21]:
 
 # Number of patients represented in the expression dataset
 clinmat_df.query("sample_id in @expr_df.index").patient_id.nunique()
 
 
-# In[21]:
+# In[22]:
 
 # Peak at the data matrix
 expr_df.iloc[:5, :5]
 
 
+# In[23]:
+
+# Save complete expression matrix
+path = os.path.join('data', 'complete', 'expression-matrix.tsv.bz2')
+expr_df.to_csv(path, sep='\t', float_format='%.3g', compression='bz2')
```

Comment:
Yeah I think it's fine, here's the [doc for general format](https://docs.python.org/3.5/library/string.html#formatspec):

> General format. For a given precision `p >= 1`, this rounds the number to p significant digits and then formats the result in either fixed-point format or in scientific notation, depending on its magnitude.

Based on my understanding, '%.3g' will corrupt a value by at most 0.1%, which seems trivial especially for noisy gene expression.